internal/nodes/dequeue_ingestion.go

```diff
@@ -658,11 +658,9 @@ func handleL4SharedVipService(namespacedVipKey, key string, fullsync bool) {
 		aviModelGraph := NewAviObjectGraph()
 		vipKey := strings.Split(namespacedVipKey, "/")[1]
 		aviModelGraph.BuildAdvancedL4Graph(namespace, vipKey, key, true)
-		if len(aviModelGraph.GetOrderedNodes()) > 0 {
-			ok := saveAviModel(modelName, aviModelGraph, key)
-			if ok && !fullsync {
-				PublishKeyToRestLayer(modelName, key, sharedQueue)
-			}
+		ok := saveAviModel(modelName, aviModelGraph, key)
```

Comment:
why do we want to remove this condition?

Reply:
Intention was to revert it back to original code base which serves the same purpose as this one.